### ISSUE TITLE
Multi-output Regression 

### ISSUE THREAD

**Issue by Pravanop** [materialsvirtuallab/megnet#313]
Can MEGNET handle multi-output regression targets?

**Comment by chc273** [comment]
@Pravanop Yes, you can try set "ntarget" in the constructor.  https://github.com/materialsvirtuallab/megnet/blob/3d4b9765c723b70567a6a370f23be2665ba39b24/megnet/models/megnet.py#L46

**Comment by pzhanggit** [comment]
Do you have the performance MAE results of multi-output regression on the benchmark datasets?  Thanks

**Comment by chc273** [comment]
@pzhanggit no, we do not

**Comment by pzhanggit** [comment]
> @pzhanggit no, we do not

Thx
